config/initializers/carrierwave.rb

```diff
@@ -7,4 +7,12 @@
     }
     config.fog_directory = ENV["S3_BUCKET"]
   end
+
+  if Rails.env.test? || Rails.env.cucumber?
+    CarrierWave.configure do |config|
```

Comment:
please make the 'config' here something other than 'config' - you've already got a local variable named config above. ('cw' would work!)